metadata.json

```diff
@@ -20,7 +20,7 @@
    ],
   "dependencies": [
     { "name": "puppetlabs/stdlib", "version_requirement": ">= 4.6.0 < 5.0.0" },
-    { "name": "puppetlabs/apt", "version_requirement": ">= 0.0.3 < 2.0.0" },
+    { "name": "puppetlabs/apt", "version_requirement": ">= 0.0.3 < 3.0.0" },
```

Comment:
I think this should be either `>= 1.0.0 < 3.0.0` or `>= 2.0.0 < 3.0.0` if we can drop support for the apt 1.x API.